### PR TITLE
Prepare release 0.101.2

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

File: doc/how_to/combine_recordings.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Combine Recordings in SpikeInterface
+Combine recordings in SpikeInterface
```
Comment (zm711) [inline doc/how_to/combine_recordings.rst:1]: We may want to add a note in the docs explaining the capitalization rules for tutorial titles.  I think first word or "important" words are both acceptable (in English at least) so if we are picking a style it should be mentioned in our style guide somewhere.

Comment (alejoe91) [reply doc/how_to/combine_recordings.rst:1]: I'll add a note. Currently we had a random style :)

File: src/spikeinterface/core/sortinganalyzer.py [hunk 2]
```diff
@@ -2245,9 +2245,15 @@ def _save_data(self):
                 elif HAS_PANDAS and isinstance(ext_data, pd.DataFrame):
                     df_group = extension_group.create_group(ext_data_name)
                     # first we save the index
-                    df_group.create_dataset(name="index", data=ext_data.index.to_numpy())
+                    indices = ext_data.index.to_numpy()
```
Comment (alejoe91) [inline src/spikeinterface/core/sortinganalyzer.py:2248]: we had a tutorial failing because of this

File: doc/how_to/combine_recordings.rst (hunk 1 above)
Comment (alejoe91) [reply doc/how_to/combine_recordings.rst:1]: done in last commit :)

File: doc/development/development.rst [hunk 3]
```diff
@@ -192,6 +192,7 @@ Miscelleaneous Stylistic Conventions
 #. Avoid using abbreviations in variable names (e.g. use :code:`recording` instead of :code:`rec`). It is especially important to avoid single letter variables.
 #. Use index as singular and indices for plural following the NumPy convention. Avoid idx or indexes. Plus, id and ids are reserved for identifiers (i.e. channel_ids)
 #. We use file_path and folder_path (instead of file_name and folder_name) for clarity.
+#. For the titles of documentation pages, only capitalize the first letter of the first word and classes or software packages. For example, "How to use a SortingAnalyzer in SpikeInterface".
```
Comment (zm711) [inline doc/development/development.rst:195]: Perfect thanks!

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**